docs/auth/oauth.md

```diff
@@ -36,16 +36,16 @@ $ http -v GET http://localhost:8080/auth/token?grant_type=client_credentials "Au
 | name                          | The unique name of your OAuth Server                                                      |
 | oauth_endpoints.authorize     | Defines the [proxy configuration](/docs/config/proxy.md) for the `authorize` endpoint     |
 | oauth_endpoints.token         | Defines the [proxy configuration](/docs/config/proxy.md) for the `token` endpoint         |
-| oauth_endpoints.info          | Defines the [proxy configuration](/docs/config/proxy.md) for the `info` endpoint          |
+| oauth_endpoints.introspection | Defines the [proxy configuration](/docs/config/proxy.md) for the `introspection` endpoint |
 | oauth_endpoints.revoke        | Defines the [proxy configuration](/docs/config/proxy.md) for the `revoke` endpoint        |
 | oauth_client_endpoints.create | Defines the [proxy configuration](/docs/config/proxy.md) for the `create` client endpoint |
 | oauth_client_endpoints.remove | Defines the [proxy configuration](/docs/config/proxy.md) for the `remove` client endpoint |
 | allowed_access_types          | The allowed access types for this oauth server                                            |
 | allowed_authorize_types       | The allowed authorize types for this oauth server                                         |
 | auth_login_redirect           | The auth login redirect URL                                                               |
 | secrets                       | A map of client_id: client_secret that allows you to authenticate only with the client_id |
-| token_strategy.name           | The token validation strategy for this server. Could be `storage` or `jwt`                |
-| token_strategy.settings       | Token strategy settings, see bellow by strategy                                           |
+| token_strategy.name           | The token strategy for this server. Could be `introspection` or `jwt`                           |
+| token_strategy.settings.secret| If you use JWT you should set your secret or private certificate string here              |
```

Comment:
This should be removed, I added more docs on this below, only `token_strategy.settings` should be here.